test/es6/default-splitscope-undodeferparse.js

```diff
@@ -0,0 +1,64 @@
+//-------------------------------------------------------------------------------------------------------
+// Copyright (C) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE.txt file in the project root for full license information.
+//-------------------------------------------------------------------------------------------------------
+
+function foo1(a, b) {
+    if (b != 10) {
+        print("FAILED")
+    } else {
+        print("PASSED");
```

Comment:
No actually RL does look at each line and if any of them is not "Passed" it will report the test as failed. So we are fine on this.